### PR TITLE
Update data-processing-tools.md

### PR DESCRIPTION
**Pull request by demkodo**
Feedback welcome, as always. Thanks

-----
[View rendered organization/data-processing-tools.md](https://github.com/publiccodenet/about/blob/add-Odoo-to-GDPR/organization/data-processing-tools.md)

### CODE REVIEW

File: organization/data-processing-tools.md [hunk 1]
```diff
@@ -31,3 +31,11 @@ We use Tentoo for human resources management, including processing salaries and
 Tentoo's [General Terms and Conditions](https://www.tentoo.nl/images/downloads/Generaltermsandconditions_TentooPartners.pdf) say that Tentoo doesn't process data outside the European Economic Area (EEA):
 
 > 6.1 Tentoo will only process the Personal Data in the European Economic Area (EEA), and will ensure that no processing actions at all take place outside the EEA, unless Tentoo has obtained the Client’s explicit, written consent to this.
+
+## Odoo
+
+We use Odoo Community for its CRM functionality and project management pipelines for our codebases and membership development.
```
Comment (ElenaFdR) [inline organization/data-processing-tools.md:37]:
```suggestion
We use Odoo Community for customer relationship management (CRM) and project management for our codebases and membership development.
```

File: organization/data-processing-tools.md [hunk 2]
```diff
@@ -31,3 +31,11 @@ We use Tentoo for human resources management, including processing salaries and
...
+
+Odoo's responsibilities in terms of personal data protection depend on their [various data processing activities](https://www.odoo.com/gdpr).
+
+> As a Data Controller, our activities are covered in our Privacy Policy, which has been updated in accordance to GDPR. This policy explains as clearly as possible what data we process, why we process it, and how we do it. Closely related to this, our Security Policy explains the security best practices we implemented at Odoo, at all levels (technical and organizational) in order to guarantee that your data is processed in a safe and secure manner.
```
Comment (ElenaFdR) [inline organization/data-processing-tools.md:41]: I've added the hyperlinks from the original.
```suggestion
> As a Data Controller, our activities are covered in our [Privacy Policy](https://www.odoo.com/privacy), which has been updated in accordance to GDPR. This policy explains as clearly as possible what data we process, why we process it, and how we do it. Closely related to this, our [Security Policy](https://www.odoo.com/security) explains the security best practices we implemented at Odoo, at all levels (technical and organizational) in order to guarantee that your data is processed in a safe and secure manner.
```

**Comment by ElenaFdR** [comment]
@ericherman That language is directly quoted from Odoo's GDPR guide.

**Comment by bvhme** [comment]
As I understand it we're not using Odoo's service but we're using their software hosted by OpusVL, doesn't this mean they are the data processor?

**Comment by demkodo** [comment]
> As I understand it we're not using Odoo's service but we're using their software hosted by OpusVL, doesn't this mean they are the data processor?

I think that is indeed what that means. Good point. I'll update the PR and link to the privacy policy of Opus VL.

File: organization/data-processing-tools.md [hunk 3]
```diff
@@ -31,3 +31,11 @@ We use Tentoo for human resources management, including processing salaries and
 Tentoo's [General Terms and Conditions](https://www.tentoo.nl/images/downloads/Generaltermsandconditions_TentooPartners.pdf) say that Tentoo doesn't process data outside the European Economic Area (EEA):
 
 > 6.1 Tentoo will only process the Personal Data in the European Economic Area (EEA), and will ensure that no processing actions at all take place outside the EEA, unless Tentoo has obtained the Client’s explicit, written consent to this.
+
+## Odoo
+
+We use Odoo Community for customer relationship management (CRM) and project management for our codebases and membership development. Opus VL is our hosting provider for Odoo Community.
+
+Opus VL's describes its [data processing activities in their privacy policy](https://opusvl.com/privacy-policy).
```
Comment (ElenaFdR) [inline organization/data-processing-tools.md:39]:
```suggestion
Opus VL's data processing activities are described in their [privacy policy](https://opusvl.com/privacy-policy).
```

Comment (ElenaFdR) [reply organization/data-processing-tools.md:39]: More urgent concern: this privacy policy is for the opusvl.com website, not for hosting provided by OpusVL the company.

Comment (bvhme) [reply organization/data-processing-tools.md:39]: So this needs to be resolved first @demkodo

**Review by bvhme — CHANGES_REQUESTED** [review]
Mentioned the wrong privacy policy.

**Comment by ElenaFdR** [comment]
I'm going to close this PR, since this content has now been moved to the new tool management and guidance folder (merged to live with #746).

**Pull request closed without merging**